tests/unit/example.test.ts

```diff
@@ -0,0 +1,7 @@
+function sum(a: number, b: number): number {
+  return a + b;
+}
+
+test("adds 1 + 2 to equal 3", () => {
+  expect(sum(1, 2)).toBe(3);
+});
```

Comment:
Do we need this example? or this was just for testing!!

Reply:
it's just a starter example replacing the deprecated one that was available in the reop. it's just for showing how jest functions work for anyone creating tests in the future.

Reply:
I think if it is related to the project will be more efficient, what o you think if we changed it to be something like.

```suggestion
function starter(): number {
  return window.config.network;
}

test("Check the selected network", () => {
  expect(starter()).toBe("dev");
});
```
also, we can change it to ensure that the selected network in the networks array like `[ main, qa, test, dev, custom]`